global/parameters.go

```diff
@@ -11,3 +11,6 @@ var ServerConfig = ServerOpts{
 	WriteTimeout: 15,
 	ReadTimeout:  15,
 }
+
+const CsvPath string = "assets/pokemons.csv"
+const ApiUrl string = "https://pokeapi.co/api/v2/"
```

Comment:
Remember about acronyms. 

Group successive const definitions:

```suggestion
const (
	CSVPath = "assets/pokemons.csv"
	APIBaseURL = "https://pokeapi.co/api/v2/" // or something similar, technically speaking it's not an URL
)
```